ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -283,13 +283,13 @@ object BigDiffy extends Command with Serializable {
     new BigDiffy[T](lhs, rhs, d, keyFn, ignoreNan)
 
   /** Diff two Avro data sets. */
-  def diffAvro[T <: GenericRecord : ClassTag : Coder](sc: ScioContext,
+  def diffAvro(sc: ScioContext,
                                               lhs: String, rhs: String,
-                                              keyFn: T => MultiKey,
-                                              diffy: AvroDiffy[T],
+                                              keyFn: GenericRecord => MultiKey,
```

Comment:
What's up with this change?

Reply:
type parameters for sc.avroFile now have SpecificRecord as a type bound

https://github.com/spotify/scio/blob/4609bd7a2fe0359ea45e60caa29e909073b9add7/scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala#L82

I had no choice but to remove the type parameters for sc.avroFile[GenericRecord], and was 80% sure this would be OK to have keyFn and and diffy not be generic types.

Reply:
is this ok if it's being used directly in a pipeline?

Reply:
We can retain the type safety here for compatibility but change how we call it in scio. GenericRecord keyFn is also likely much weirder to deal with